sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -4421,348 +4421,1123 @@
+                              "redrawOn": "",
+                              "modalEdit": false,
+                              "dataGridLabel": false,
+                              "labelPosition": "top",
+                              "description": "",
+                              "errorLabel": "",
+                              "tooltip": "",
+                              "hideLabel": false,
+                              "tabindex": "",
+                              "disabled": false,
+                              "autofocus": false,
+                              "dbIndex": false,
+                              "customDefaultValue": "",
+                              "calculateValue": "",
+                              "calculateServer": false,
+                              "widget": {
+                                "type": "input"
+                              },
+                              "attributes": {},
+                              "validateOn": "change",
+                              "conditional": {
+                                "show": null,
+                                "when": null,
+                                "eq": ""
+                              },
+                              "overlay": {
+                                "style": "",
+                                "left": "",
+                                "top": "",
+                                "width": "",
+                                "height": ""
+                              },
+                              "allowCalculateOverride": false,
+                              "encrypted": false,
+                              "showCharCount": false,
+                              "showWordCount": false,
+                              "properties": {},
+                              "allowMultipleMasks": false,
+                              "addons": [],
+                              "mask": false,
+                              "inputType": "text",
+                              "inputFormat": "plain",
+                              "inputMask": "",
+                              "displayMask": "",
+                              "spellcheck": true,
+                              "truncateMultipleSpaces": false,
+                              "id": "e4ptvz"
+                            }
+                          ],
+                          "width": 6,
+                          "offset": 0,
+                          "push": 0,
+                          "pull": 0,
+                          "size": "md",
+                          "currentWidth": 6
+                        },
+                        {
+                          "components": [
+                            {
+                              "label": "Course code",
+                              "tooltip": "Your course code is also known as your course registration number typically shown beside your course name (e.g. MGMT 101)",
```

Comment:
It seems that there are some special characters in the middle of the string. Was it expected?
![image](https://user-images.githubusercontent.com/61259237/177859192-f4a5af96-5bfd-45e1-b44b-1bb102fa0e7a.png)

Reply:
copied from figma, the words. Might have some special characters from them. Removed it now 👍